src/RulesEngine/Models/Workflow.cs

```diff
@@ -1,25 +1,36 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 
 namespace RulesEngine.Models
 {
+    [Obsolete("Workflow is now Workflow - Workflow will be removed in next major version")]
```

Comment:
I believe you meant - "WorkflowRules is now called Workflow. WorkflowRules will be removed in next major version."

Reply:
A better message that i can think of : - "WorkflowRules class is deprecated. Use Workflow class instead"